torch/csrc/jit/runtime/register_distributed_ops.cpp

```diff
@@ -90,7 +105,8 @@ RegisterOperators reg_rpc_ops({
              // `kwargs = kwargs if kwargs is not None else {}`.
              auto& kwargsDictIValue =
                  num_inputs >= 4 ? *stackIter++ : emptyDict;
-             TORCH_INTERNAL_ASSERT(dstWorkerNameIValue.isString());
+             TORCH_INTERNAL_ASSERT(
+                 dstWorkerNameIValue.isString() || dstWorkerNameIValue.isInt());
```

Comment:
Since it can be an int, we should call it `dstWorkerIValue` instead of `dstWorkerNameIValue`?

Reply:
Right, better name.